pe/x86/64/mingw32-zlib/fedora-zlib.description

```diff
@@ -0,0 +1 @@
+MinGW ZLib build Version 1.3
```

Comment:
The name of this file is wrong. it should be `mingw32-zlib/mingw32-zlib.description` also why mingw32 when is a 64 bit?

Reply:
The compiler is named `x86_64-mingw32-gcc`
I think there is no mingw64